dluhc-component-library/src/components/datasets/DatasetList.tsx

```diff
@@ -1,12 +1,12 @@
 import { useCallback } from "preact/hooks";
+import { Dataset } from "src/api/planningData/types";
 import DatasetSelect from "./DatasetSelect";
-import { Dataset } from "src/api/types";
 
 interface DatasetListProps {
   title?: string;
   items: Dataset[];
-  selectedItems: Dataset[];
-  onSelect: (dataset: Dataset) => void;
+  selectedItems: string[];
```

Comment:
`Non-blocker` Its generally better to type these as `ReadonlyArray<string>` so we dont accidently mutate arrays we shouldn't